src/arch/riscv/isa.cc

```diff
@@ -267,6 +267,11 @@ ISA::ISA(const Params &p) :
     _regClasses.push_back(&ccRegClass);
     _regClasses.push_back(&miscRegClass);
 
+    fatal_if( p.vlen < p.elen,
+    "VLEN should be greater or equal",
+        "than ELEN. Ch. 2RISC-V vector spec.");
+
```

Comment:
Maybe we can add here the message @hnpl suggested:

inform("RVV enabled, VLEN = %d bits, ELEN = %d bits", p.vlen, p.elen)

Reply:
I agree that it is a good place to put it. Added in the last update